lib/admin-icon-preview.php

```diff
@@ -171,6 +175,11 @@ function get_icon_output( $location, $icons = array() ){
 				
 					$data_title = 'Pin';
 					break;
+
+				case 'love' :
+				
+					$data_title = __('Love', 'genesis');
```

Comment:
Looks like I need to actually make these all translatable strings, but that will have to be done separately. However, the correct text domain should be `genesis-simple-share` not `genesis`.